### PR TITLE
Extract methods from consoles::sshVirtsh::add_disk

### PR DESCRIPTION
**Pull request by okurz**
* consoles::sshVirtsh: Extract methods for copy_image variants
* consoles::sshVirtsh: Extract methods for driver_elem/handle_disk_type/bootorder_elem
* consoles::sshVirtsh: Extract methods for create_disk/copy_image
* Use signatures in consoles::sshVirtsh

### CODE REVIEW

**Comment by okurz** [comment]
merge is currently blocked due to https://progress.opensuse.org/issues/97700 waiting for https://build.opensuse.org/request/show/914981

File: consoles/sshVirtsh.pm [hunk 1]
```diff
@@ -16,12 +16,8 @@
 
 package consoles::sshVirtsh;
 
-use strict;
-use warnings;
+use Mojo::Base 'consoles::sshXtermVt', -strict, -signatures;
```
[the remark this answers is not in the document]
Comment (kraih) [reply consoles/sshVirtsh.pm:19]: It should be fine. Just the `-strict` does nothing here.

**Review by Martchus — APPROVED** [review]
It looks like many of these functions are directly called by the test distribution. However, I don't see any uses which would cease to work.

**Comment by jlausuch** [comment]
Looks like this breaks some tests: https://openqa.suse.de/tests/7099572#step/patch_and_reboot/95

**Comment by Martchus** [comment]
@jlausuch This PR should fix it: https://github.com/os-autoinst/os-autoinst/pull/1771

**Comment by jlausuch** [comment]
> @jlausuch This PR should fix it: #1771

Thank you for the fast reaction :)

**Comment by mloviska** [comment]
https://openqa.suse.de/tests/7126875#step/bootloader_svirt/31 -> https://github.com/os-autoinst/os-autoinst/pull/1774

**Pull request merged**